my-react-study-app/src/widgets/detailedSection/detailedItemsControls.css

```diff
@@ -0,0 +1,57 @@
+.main__detailed_items_controls {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    gap: 20px;
+    width: 100%;
+    padding: 10px;
+    background-color: var(--beige);
+    position: fixed;
+    z-index: 50;
+    bottom: 0;
+    left: 0;
+    &.light {
+        background-color: var(--beige);
+    }
+    &.dark {
+        background-color: var(--deep-blue);
+    }
+}
+
+.detailed_items_controls__text {
+    font-family: var(--font-family);
+    font-size: 1.6rem;
+    &.light {
+        color: var(--deep-blue);
+    }
+    &.dark {
+        color: var(--beige);
+    }
+}
+
+.detailed_items_controls__button {
+    display: flex;
+    width: 130px;
+    height: 40px;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    font-size: 14px;
+    text-transform: uppercase;
+    transition: all 0.2s;
+    &.light {
+        background-color: var(--deep-blue);
+        color: var(--white);
+    }
+    &.light:hover {
+        background-color: var(--electric-orange);
+    }
+    &.dark {
+        background-color: var(--turquoise);
+        color: var(--white);
+    }
+    &.dark:hover {
+        background-color: var(--electric-orange);
+    }
```

Comment:
you can group repeated styles:
` &.dark:hover,
  &.light:hover {
        background-color: var(--electric-orange);
    }`